firedrake/interpolation.py

```diff
@@ -113,6 +113,13 @@ class Interpolator(object):
         interpolation over.
     :kwarg freeze_expr: Set to True to prevent the expression being
         re-evaluated on each call.
+    :kwarg access: The access descriptor for combining updates to shared dofs.
+    :kwarg bcs: An optional list of boundary conditions to zero-out in the
+        output function space. Interpolator rows or columns which are
+        associated with boundary condition nodes are zeroed out when this is
+        specified.
+    :kwarg allow_missing_points: For interpolation across meshes: allow
+        points in the target mesh that are not in the source mesh.
```

Comment:
What happens when this is the case? And if `False` I expect that a particular exception is raised.

Reply:
At the moment you get something like
```
ValueError: 117 vertices are outside the mesh and have been removed from the VertexOnlyMesh
```
Is this sufficient or do I need to try and catch it then rethrow with something more specific?

Reply:
I've decided to rethrow something more specific.

Reply:
My point was that the docstring was unclear. You should just include something like "If `False` then an exception will be raised if points are located outside the source mesh". A more specific exception sounds good though.

Reply:
See the updated docstring